### PR TITLE
Add collections cache disabling to values.yaml

### PR DESCRIPTION
**Pull request by nikovacevic**
## What does this PR change?
Adds commented out values for disabling collections memory cache.

## Does this PR rely on any other PRs?
No

## How does this PR impact users? (This is the kind of thing that goes in release notes!)
N/A

## Links to Issues or tickets this PR addresses or fixes
N/A

## What risks are associated with merging this PR? What is required to fully test this PR?
N/A

## How was this PR tested?
N/A

## Have you made an update to documentation? If so, please provide the corresponding PR.
N/A

### CODE REVIEW

File: cost-analyzer/values.yaml [hunk 1]
```diff
@@ -2258,6 +2258,11 @@ kubecostAggregator:
   # `deployMethod: "statefulset"`
   replicas: 1
 
+  # Collections queries are cached in memory. Turning it off is not recommended
```
Comment (chipzoller) [inline cost-analyzer/values.yaml:2261]: Can you explain what collections are very simply?

Comment (nikovacevic) [reply cost-analyzer/values.yaml:2261]: @kwombach12 care to weigh in? Haha.

**Comment by jessegoodier** [comment]
@nikovacevic do we need this for 2.0? Happy to help here.

**Comment by chipzoller** [comment]
Conflicts here.

**Comment by nikovacevic** [comment]
Totally up to you guys. This was a request from @chipzoller.

**Comment by chipzoller** [comment]
Conflicts

**Comment by chipzoller** [comment]
Converted to draft until resolved.

**Pull request closed without merging**